Sources/Extensions/Foundation/ArrayExtensions.swift

```diff
@@ -34,8 +35,10 @@ public extension Array where Element: FloatingPoint {
 	///
 	/// - Returns: average of the array's elements.
 	public func average() -> Element {
-		// http://stackoverflow.com/questions/28288148/making-my-function-calculate-average-of-array-swift
-		return isEmpty ? 0 : reduce(0, +) / Element(count)
+        guard isEmpty == false else { return 0 }
```

Comment:
I actually specifically don't use `!` because I don't think its readable at all 🤔 
I only use it to invert `Booleans` when setting values. Not when writing conditionals.
We can still change it though